### PR TITLE
update friend name from ark to velero

### PR DESCRIPTION
**Pull request by cpanato**
- update Ark to use the brand new name Velero
- also update the Velero logo

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ kubicorn/docs/
 ├── assets/
 │ → This folder basically contains the favicons. We should probably move this
 │   to img/ sometime.
-│   
+│
```
Comment (frank-theunicorn) [inline docs/README.md]: Nit... but... Meeps Meeps... this looks weird...

File: docs/_friends/velero.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ---
-title: Ark
-logo: /img/ark.png
+title: Velero
+logo: /img/velero.png
```
Comment (frank-theunicorn) [inline docs/_friends/velero.md]: Hi, just checking in. Wondering why this is here...? 🌈🐼

File: docs/_friends/velero.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
...
 ---
 
-Thanks to the fantastic works from our friends at Heptio, we can easily help you snapshot and recreate your entire `kubernetes` platform using Heptio's [Ark](https://github.com/heptio/ark).
\ No newline at end of file
+Thanks to the fantastic works from our friends at Heptio, we can easily help you snapshot and recreate your entire `kubernetes` platform using Heptio's [Velero](https://github.com/heptio/velero).
```
Comment (frank-theunicorn) [inline docs/_friends/velero.md]: Hi...🦄 So I read your PR... Woah... this old bit again... 🦄

File: docs/README.md [hunk 4]
```diff
@@ -86,7 +86,7 @@ Simply update the associated `.md` markdown file. All documentation should be in
 
 # Testing changes
 
-If you have Jekyll stack installed you can run the website locally to test changes. To install Jekyll, follow the [Setting up your GitHub Pages site locally with Jekyll](https://help.github.com/articles/setting-up-your-github-pages-site-locally-with-jekyll/) tutorial.  
+If you have Jekyll stack installed you can run the website locally to test changes. To install Jekyll, follow the [Setting up your GitHub Pages site locally with Jekyll](https://help.github.com/articles/setting-up-your-github-pages-site-locally-with-jekyll/) tutorial.
```
Comment (frank-theunicorn) [inline docs/README.md]: Hi, just checking in. Meeps Meeps... this looks weird...

File: docs/README.md [hunk 5]
```diff
@@ -16,7 +16,7 @@ kubicorn/docs/
 ├── assets/
 │ → This folder basically contains the favicons. We should probably move this
 │   to img/ sometime.
-│
```
Comment (frank-theunicorn) [inline docs/README.md]: Hi, just checking in. Frank is confused... 🐼

File: docs/README.md [hunk 6]
```diff
@@ -86,7 +86,7 @@ Simply update the associated `.md` markdown file. All documentation should be in
 
 # Testing changes
 
-If you have Jekyll stack installed you can run the website locally to test changes. To install Jekyll, follow the [Setting up your GitHub Pages site locally with Jekyll](https://help.github.com/articles/setting-up-your-github-pages-site-locally-with-jekyll/) tutorial.
```
Comment (frank-theunicorn) [inline docs/README.md]: Me again, just one more nit...🌈 Something looks weird here. Are you sure about this? 🌈

File: docs/_friends/velero.md [hunk 7]
```diff
@@ -1,6 +1,6 @@
 ---
-title: Ark
-logo: /img/ark.png
+title: Velero
```
Comment (frank-theunicorn) [inline docs/_friends/velero.md]: Hrmmm... Hrmm.. this looks funny...

**Comment by cpanato** [comment]
cc @kris-nova @xmudrii

File: docs/_friends/velero.md (hunk 7 above)
Comment (cpanato) [reply docs/_friends/velero.md]: lol

File: docs/_friends/velero.md (hunk 3 above)
Comment (detiber) [inline docs/_friends/velero.md]:
```suggestion
Thanks to the fantastic works from our friends at VMware, we can easily help you snapshot and recreate your entire `kubernetes` platform using [Project Velero](https://github.com/heptio/velero).
```

Comment (cpanato) [reply docs/_friends/velero.md]: :)

Comment (cpanato) [reply docs/_friends/velero.md]: @detiber done, thanks for the review

**Pull request merged**